sdk/identity/azure-identity/azure/identity/_credentials/azure_cli.py

```diff
@@ -61,6 +62,10 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=no-self-use,unused-arg
 
         return token
 
+    @property
+    def token_refresh_offset(self):
```

Comment:
The feature doesn't make sense for this credential because it doesn't cache tokens.

Reply:
This is to implement the contract. 

it defaults to 120s just tells policy to not set the offset less than 120s which does not do harm.

Reply:
Btw, as we discussed, it is only useful if it is greater than 300.

Reply:
The contract will be a protocol credentials won't be required to implement. Why add to this credential a public property returning misleading values?

Reply:
Fair enought